### PR TITLE
set MIG as deprecated

### PR DESCRIPTION
**Pull request by dernorberto**
Mozilla MIG has been deprecated and the URL is wrong

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,30 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
...
-- [OSX Evidence Collection](#osx-evidence-collection)
-- [Other lists](#other-lists)
-- [Other tools](#other-tools)
-- [Playbooks](#playbooks)
-- [Process Dump Tools](#process-dump-tools)
-- [Sandboxing/reversing tools](#sandboxingreversing-tools)
-- [Timeline tools](#timeline-tools)
-- [Videos](#videos)
-- [Windows Evidence Collection](#windows-evidence-collection)
+- [Awesome Incident Response ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-incident-response-img-src%22httpsgithubcomsindresorhusawesome%22-alt%22awesome%22)
```
Comment (meirwah) [inline README.md]: what is this change related to ?

Comment (PolarBearGod) [reply README.md:9]: Looks like they accidentally deleted some lines by accident?

- [Contents] (#contents)
- [Books] (#books)	  
- [IR tools Collection] (#ir-tools-collection)

**Comment by meirwah** [comment]
@dernorberto can you fix PR to include only the MIG relevant changes ?

**Comment by Karneades** [comment]
The fix for URL is great, that helps.

To be a little bit nit-picky regarding the part with the deprecation notice; an awesome list should avoid having deprecated items. To [cite](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md#requirements-for-your-awesome-list):
> Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.

So, either removing the entry (or put it into a separate file) if we think the deprecated tool should be avoided at all or avoid using the notice and leave it to the reader finding that out on the MIG page because it still works and users should include that tool in their evaluation too.

**Comment by meirwah** [comment]
@dernorberto will reopen once you fix comments
